eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -68,6 +68,26 @@ jobs:
 
       - script: |
           node common/scripts/install-run-rush.js install
+        displayName: 'Install Rush'
+
+      - script: |
+          node common/scripts/install-run-rush.js lint $(rushFlags)
+        displayName: 'Run lint'
```

Comment:
Similarly let’s call this “Lint libraries”

Reply:
done